packages/devtools/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ description: A suite of web-based performance tooling for Dart and Flutter.
 # package remaining purely as a container for the compiled copy of the devtools
 # app. That ensures that version constraints for the devtools_app do not impact
 # this package.
-version: 0.2.2
+version: 0.2.3
```

Comment:
please hold off on changing the devtools version numbers throughout this CL. I have a separate [PR](https://github.com/flutter/devtools/pull/1689) that updates the version numbers (we are doing another release this week). Go ahead and leave the changelog updates -  I will handle the merge conflicts on my CL